arrow-site/doc/docs/arrow-core/datatypes/sum/README.md

```diff
@@ -33,7 +33,7 @@ We can change the active side with `changeSide`.
 sum.changeSide(Sum.Side.Right).extract(Store.comonad(), Store.comonad())
 ```
 
-As Sum is also a `Comonad` we have access to `coflatmap` and `map` for changing the `extract` result.
+As Sum is also a `Comonad`, we have access to `coflatmap` and `map` for changing the `extract` result.
```

Comment:
`Sum`

Okay. I'm going to stop here, but this is a fantastic improvement in the documentation. Thanks so much for this hard work. Instead of making suggestions here, I'll file a PR at a later time.

Reply:
You bet! And thanks for looking over these. You caught some great things I missed.